mobify/astro-scaffold
# Add UAT build configuration to scaffold

Adds a UAT build configuration to the scaffold so new builds will have it setup when they generate

Status: **Ready For Review**

Reviewers: @jansepar @MikeKlemarewski @jeremywiebe @lizcross @jvaill @jvoll @kschmidtdev 
Linked PRs: **https://github.com/mobify/astro/pull/246**
## Changes
- Adds a UAT build configuration to scaffold
- Uses the Astro pointer in PR246 that adds build configurations to Astro framework
## How to test-drive this PR
- Checkout code
- Run scaffold app
- Ensure UAT build configuration is added
- Ensure scaffold builds and runs with UAT configuration set
## TODOS:
